app/views/alpacas/show.html.erb

```diff
@@ -0,0 +1,107 @@
+            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+            <span class="visually-hidden">Previous</span>
+          </button>
+          <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide="next">
+            <span class="carousel-control-next-icon" aria-hidden="true"></span>
+            <span class="visually-hidden">Next</span>
+          </button>
+        </div>
+
+        <div class="card-body">
+          <h5 class="card-title"><%= @alpaca.name %></h5>
+          <p class="card-text">My nickname is <%= @alpaca.nick_name %>. I'm would like to get hired by you, but I need my personal space.</p>
+          <table class="table">
+            <thead>
+              <tr>
+                <th scope="col"></th>
+                <th scope="col"></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <th scope="row">Age</th>
+                <td> <%= @alpaca.age %> years </td>
+              </tr>
+              <tr>
+                <th scope="row">Weight</th>
+                <td> <%= @alpaca.weight %> kg</td>
+              </tr>
+              <tr>
+                <th scope="row">Height</th>
+                <td> <%= @alpaca.height %> cm</td>
+              </tr>
+              <tr>
+                <th scope="row">Wool Type</th>
+                <td> <%= @alpaca.wool_type %> </td>
+              </tr>
+              <tr>
+                <th scope="row">Color</th>
+                <td> <%= @alpaca.color %> </td>
+              </tr>
+              <tr>
+                <th scope="row">Owner</th>
+                <td><%= @alpaca.owner.first_name %> <%= @alpaca.owner.last_name  %></td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+      <div class="card border-0 m-auto col-lg-6 col-sx-8 mt-4">
+        <div class="card border-0 bg-dark col-6 mt-4 m-auto">
+          <img src="https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/background.png" class="card-img rounded" alt="...">
+          <div class="card-img-overlay d-flex justify-content-center align-items-center">
+            <h5 class="text-white">Eating Grass</h5>
+          </div>
+        </div>
+        <div class="card border-0 bg-dark col-6 mt-3 mb-3 m-auto">
+          <img src="https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/background.png" class="card-img rounded" alt="...">
+          <div class="card-img-overlay d-flex justify-content-center align-items-center">
+            <h5 class="text-white">Producing Wool</h5>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="bg-white p-3 sticky-top m-auto col-lg-3 mt-3">
+
+      <div class="d-flex justify-content-between mt-3">
+        <p><strong>Price per Day</strong></p>
+        <p>£4.95</p>
```

Comment:
The total of the prices for each activities might not match the total price we put when we create an alpaca but I guess it's not too important for now...